stdlib/Random/src/misc.jl

```diff
@@ -343,6 +343,12 @@ Construct a random cyclic permutation of length `n`. The optional `rng`
 argument specifies a random number generator, see [Random Numbers](@ref).
 The element type of the result is the same as the type of `n`.
 
+Here, a "cyclic permutation" means that all of the elements lie within
+a single cycle.  If `n > 0`, there are ``(n-1)!`` possible cyclic permutations,
+which are sampled uniformly.  If `n == 0`, `randcycle` returns an empty array.
```

Comment:
```suggestion
which are sampled uniformly.  If `n == 0`, `randcycle` returns an empty vector.
```
better to be more precise when possible